Documentation/gettingstarted.rst

```diff
@@ -1042,39 +1040,37 @@ You can observe the newly created endpoints in Cilium, similar to the following
     33115      Disabled      257        mesos:id=goodclient           f00d::a00:20f:0:815b   10.15.220.6    ready
     64189      Disabled      258        mesos:id=badclient            f00d::a00:20f:0:fabd   10.15.152.27   ready    
 
-Marathon runs the tasks as batch jobs with ``stdout`` available under ``/var/lib/mesos``. To simplify the retrieval of the ``stdout`` log, use the ``tail_client.sh`` script to output each of the client logs. Tail the *goodclient* logs:
+Marathon runs the tasks as batch jobs with ``stdout`` available under ``/var/lib/mesos``. To simplify the retrieval of the ``stdout`` log, use the ``tail_client.sh`` script to output each of the client logs. In a new terminal, go to ``/cilium/examples/mesos``, start a new ssh session to the Vagrant VM with ``vagrant ssh`` and tail the *goodclient* logs:
```

Comment:
Same with the path here.

Reply:
"with stdout available under /var/lib/mesos."   'available under' sounds funny to me.  Do we me "with stdout logged to a task-specific file located in /var/lib/mesos?"  Or something else?

Reply:
I can reword it.